### PR TITLE
Created Map, Added Legend with Layer Selection for Store Locations

### PR DESCRIPTION
**Pull request by gquirk3**
- Created Map using Leaflet
- Added Legend with Layer Control
- Added Store Locations

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -6,13 +6,16 @@ import RecipePage from './RecipePage';
 
 const App: React.FC = () => {
   return (
+    
     <Router>
-      <Routes>
-        <Route path="/" element={<LaunchPage/>} />
```
Comment (ptobey) [inline src/App.tsx]: Test comment

**Review by ptobey — COMMENTED** [review]
Test comment

File: src/App.tsx (hunk 1 above)
Comment (gquirk3) [reply src/App.tsx]: i have no idea what you are talking about bro

File: public/publix.png
Comment (wreckless-omni) [inline public/publix.png:1]: Logo is great!

**Review by batman6989 — APPROVED** [review]
Looks good Man

**Review by wreckless-omni — APPROVED** [review]

**Review by icesario2 — APPROVED** [review]

**Review by laurenAcosta00 — APPROVED** [review]
Looks great!

**Pull request merged**